src/components/Profile/Sidebar.js

```diff
@@ -0,0 +1,136 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Form } from 'semantic-ui-react';
+import { connect } from 'react-redux';
+
+// import firebase
+import firebase from '../../firebase/config';
+
+// import action creator for update profile
+import {
+  updateProfileRequest,
+} from '../../store/actions/profileActions';
+
+// styles
+import './Sidebar.scss';
+
+class Sidebar extends Component {
+  onChange = (event) => {
+    /*
+      Because we named the inputs to match their
+      corresponding values in state, it's
+      super easy to update the state
+    */
+    this.setState({ [event.target.name]: event.target.value });
+  }
+
+  uploadImage(files) {
+    const fileload = firebase
+      .storage()
+      .ref(`images/${files[0].name}`)
+      .put(files[0]);
+
+    fileload.then(() => {
+      firebase
+        .storage()
+        .ref(`images/${files[0].name}`)
+        .getDownloadURL()
+        .then((url) => {
+          const image = {
+            image: url,
+          };
+          const { authenticatedUsername } = this.props;
+          const data = { profile: image };
+          // eslint-disable-next-line react/destructuring-assignment
```

Comment:
I wonder why you suppressed the ES linter here when you could have simply destructured.
```
const { updateProfileRequest } = this.props;
```

Reply:
Thanks, this change is already implemented